### DISCUSSION TITLE
Updating 'Request types' Venn Diagram

### DISCUSSION THREAD

**Discussion by Selkubi** [TU-Delft-DCC/dcc_app#62] [Announcements]
With 4 focus points, the old diagram doesn't work as well as before.
 (i) we either keep it as it was with 3 focus points (data, software, workflows)
 (ii) find other visualisations. 

Or if you trust my judgement, I can just move forward with whatever looks best.

**Comment by mwakok** [comment]
I would keep the current visualization and review the entire dashboard when we consider moving it to Tableau.

**Comment by alwil** [comment]
Sorry to snoop around ( still get updates and got curious) . When reviewing the content/migrating to Tableau,  consider unifying labeling of faculties and support rounds:

![image](https://github.com/user-attachments/assets/50c043d3-4e38-486e-89a5-4fffdf29229b)

Bon Courage!

**Comment by edvinskis** [comment]
Would also second option (i).
